app.js

```diff
@@ -2,7 +2,7 @@ const express = require('express');
 const path = require('path');
 const routeLogin = require('./routes/login');
 const routeList = require('./routes/celebrityList');
-const routeFavourite = require('./routes/favourite');
```

Comment:
I think you have to change filename
U were using lower case for filenames. Be consistent